x-pack/plugins/ml/public/application/datavisualizer/stats_table/hooks/use_data_viz_chart_theme.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import darkTheme from '@elastic/eui/dist/eui_theme_dark.json';
+import lightTheme from '@elastic/eui/dist/eui_theme_light.json';
+import type { PartialTheme } from '@elastic/charts';
+import { useUiSettings } from '../../../contexts/kibana';
+export const useDataVizChartTheme = (): PartialTheme => {
+  const IS_DARK_THEME = useUiSettings().get('theme:darkMode');
```

Comment:
not sure if we need to follow this naming convention for such constants 
```suggestion
  const isDarkTheme = useUiSettings().get('theme:darkMode');
```

Reply:
Updated here https://github.com/elastic/kibana/pull/87598/commits/bccd4e7ecfdb4f789e60f9ca14d909af97da1b61